src/com/facebook/buck/swift/SwiftDescriptions.java

```diff
@@ -70,4 +71,15 @@ private SwiftDescriptions() {
     output.bridgingHeader = args.bridgingHeader;
   }
 
+  static boolean isSharedRequested(Linker.LinkableDepType type) {
```

Comment:
I wonder if we can just do the comparison check: `type == SHARED` instead of this, but I wanted to keep the old code here. I would prefer to use the comparison instead of having a method/attribute in the enum.